core/soong_gen.go

```diff
@@ -286,6 +286,20 @@ func (m *genBackend) GenerateAndroidBuildActions(ctx android.ModuleContext) {
 	m.buildInouts(ctx, args)
 }
 
+func (m *genBackend) AndroidMkEntries() android.AndroidMkEntries {
+	// we skip if multiple outputs defined, as AndroidMkEntries struct support only single one
+	// (transform_source and generate_source basically)
```

Comment:
Can we make this clearer by replacnig the bracketed text with "This should only affect bob_transform_source and bob_generate_source"